compat/xis.h

```diff
@@ -9,11 +9,9 @@
 #ifndef _SQUID_COMPAT_XIS_H
 #define _SQUID_COMPAT_XIS_H
 
-#if HAVE_CTYPE_H
-#include <ctype.h>
-#endif
-
 #if __cplusplus
```

Comment:
The lines removed above should be _moved_ to the `#else` clause below, right?

Reply:
@kinkie, it looks like this small change request was overlooked when requesting my review. FWIW, I intend to approve this PR once this change request is resolved and the Perl script is [dropped](https://github.com/squid-cache/squid/pull/1314/#r1148711820).

Reply:
both done. Thanks!